packages/xod-arduino/src/Directives.re

```diff
@@ -122,6 +122,8 @@ let isNodeIdEnabled = code =>
   |. Code.lastPragmaEndis("nodeid")
   |. Endis.toBoolean(Code.doesReferSymbol("getNodeId", code));
 
+let doesRaiseErrors = Code.doesReferSymbol("raiseError");
```

Comment:
For consistency should take `#pragma XOD error_raise` as the first source of truth with fallback to `doesReferSymbol`

See `isNodeIdEnabled` and `areTimeoutsEnabled`